src/server/src/main/java/io/cassandrareaper/jmx/JmxProxyImpl.java

```diff
@@ -790,12 +821,15 @@ public void handleNotification(Notification notification, Object handback) {
     // we're interested in "repair"
     String type = notification.getType();
     LOG.debug(
-        "Received notification: {} with type {} and repairStatusHandler {}", notification, type, repairStatusHandler);
-    if (repairStatusHandler.isPresent() && ("repair").equals(type)) {
+        "Received notification: {} with type {} and repairStatusHandler {}",
+        notification,
+        type,
+        repairStatusHandlers);
```

Comment:
Should it print the whole map? Maybe you don't get the command id, so you can't do anything else? What's the value of printing the handler anyway?

Reply:
Ah I see you get `repairNo` in two different ways depending on API. Maybe log in those functions?

Reply:
Logging the full map doesn't make sense indeed.
I've followed your recommendation and logged in `processOldApiNotification` and `processNewApiNotification` using the repair number.